.github/workflows/pull-evaluation-tests.yaml

```diff
@@ -10,14 +10,21 @@ run-name: "Evaluation tests"
 
 on:
   pull_request_target:
+    types: [labeled, opened, edited, synchronize, reopened, ready_for_review]
```

Comment:
Can we delete `labeled` und `edited` event types? 
Labels: It seems the evaluation tests are triggered if we add/delete labels. 
Edit: triggers if description, title any other metadata is edited.

Reply:
labeled is required: https://github.com/kyma-project/kyma-companion/blob/6db22e4dfcf4562173ed08d505d08c962795b756/.github/workflows/pull-evaluation-tests.yaml#L53